src/react-components/ui-root.js

```diff
@@ -490,24 +485,10 @@ class UIRoot extends Component {
     });
   };
 
-  hasGrantedMicPermissions = async () => {
-    if (this.state.requestedScreen) {
-      // There is no way to tell if you've granted mic permissions in a previous session if we've
-      // already prompted for screen sharing permissions, so we have to assume that we've never granted permissions.
-      // Fortunately, if you *have* granted permissions permanently, there won't be a second browser prompt, but we
-      // can't determine that before hand.
-      // See https://bugzilla.mozilla.org/show_bug.cgi?id=1449783 for a potential solution in the future.
-      return false;
-    } else {
-      // If we haven't requested the screen in this session, check if we've granted permissions in a previous session.
-      return (await grantedMicLabels()).length > 0;
-    }
-  };
```

Comment:
@brianpeiris can you take a look and let me know if we owe work somewhere related to these concerns? I am happy to either fix things up elsewhere or at least migrate this comment somewhere else where it belongs.